books/RayTracingTheNextWeek.html

```diff
@@ -94,10 +93,35 @@
 </div>
 
 
+Managing Time
+--------------
+Before continuing, let's think about time, and how we might manage it across one or more renders.
+There are two aspects of shutter timing to think about: the time from one shutter opening to the
+next shutter opening, and how long the shutter stays open for each frame. Standard movie film used
+to be shot at 24 frames per second. Modern digital films can be 30 or even 60 frames per second.
```

Comment:
"Modern films are only limited by the benevolence of their directors, and can be found in 30, 60, 120, or even 48 frames per second. Yes! Even 48 frames per second, who would have thought of it!"

Something like that.

Reply:
Did a bunch of rewording.